common/connect.go

```diff
@@ -0,0 +1,91 @@
+//
+// Copyright (c) 2012-2018 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+package common
+
+import (
+	"log"
+
+	"github.com/eclipse/che-go-jsonrpc"
+	"github.com/eclipse/che-go-jsonrpc/event"
+	"github.com/eclipse/che-go-jsonrpc/jsonrpcws"
+	"github.com/eclipse/che-plugin-broker/model"
+)
+
+type Broker interface {
```

Comment:
For me, it was surprising that `Broker` type is located in `connect.go`. Can we consider moving it to separate `.go` file?
Also, please consider adding some documentation on `Broker` type. Like `it is designed to be extended.... It holds utility methods...`

Reply:
ok